src/PatientTestsApi/Test/Services/TestDataService.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { DBFixture } from "../Fixtures/DBFixture";
+import { expect } from "chai";
+import { TestDataService } from "../../Services/TestDataService";
+import { TestFixture } from "../Fixtures/TestFixture";
+
+const db = new DBFixture();
+
+describe("TestDataService #integration", async function (): Promise<void> {
+  before(async function (): Promise<void> {
+    await db.init();
+    await db.cleanTests();
+  });
+
+  
+  it("Can create a test", async function (): Promise<void> {
+    const dataService: TestDataService = createTestDataService();
+    const expectedTest = TestFixture.createTest();
+    expectedTest.id = "newId";    
+    const id = await dataService.insertTest(expectedTest);
+
+    // test via standard db query to see first if the record is consistent
+    const createdTest = await db.loadTest(id);
+    Object.keys(expectedTest).forEach(key => {
+      expect(createdTest[key]).deep.equal(expectedTest[key]);
+    });
+    expect(createdTest._id).is.equal(id);
+    expect(createdTest._shardKey).is.equal(expectedTest.patientId);
+
+    // // test finding patient via data service
```

Comment:
This test will be added in another ticket, right?

Reply:
yep, working on it now, as part of #3